cordyceps/cordyceps/vs_manager.py

```diff
@@ -94,13 +94,13 @@ def construct_mock_task(self) -> Task:
 
         task = Task()
         start_pose = Pose()
-        start_pose.position.x = 0.0
-        start_pose.position.y = 0.0
+        start_pose.position.x = 2.0
+        start_pose.position.y = 2.0
         goal_pose = Pose()
         task.start_pose = start_pose
         task.goal_pose = goal_pose
-        task.number_of_robots = 2
-        task.diameter = 1
+        task.number_of_robots = 8
```

Comment:
```suggestion
        task.number_of_robots = 4
```

```suggestion
        task.number_of_robots = 3
```

You can also put it on 3